serve_event.go

```diff
@@ -0,0 +1,117 @@
+package wiremock
+
+type ServeEvents struct {
+	ServeEvents []ServeEvent `json:"serveEvents,omitempty"`
+}
+
+type ServeEvent struct {
```

Comment:
This type is used by the `GetRequest*` calls, and by the `DeleteRequestsBy` call, but they're both different response. While this works, it's not good practice to have this type that has two purposes.